packages/fxa-settings/src/pages/ResetPasswordRedesign/ResetPasswordStart/container.tsx

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RouteComponentProps, useLocation, useNavigate } from '@reach/router';
+import React, { useState } from 'react';
+import { getLocalizedErrorMessage } from '../../../lib/auth-errors/auth-errors';
+import { useAuthClient, useFtlMsgResolver } from '../../../models';
+
+import { ResetPasswordStartContainerProps } from './interfaces';
+import { queryParamsToMetricsContext } from '../../../lib/metrics';
+import ResetPasswordStart from '.';
+
+const ResetPasswordStartContainer = ({
+  integration,
+  flowQueryParams = {},
+  serviceName,
+}: ResetPasswordStartContainerProps & RouteComponentProps) => {
+  const authClient = useAuthClient();
+  const ftlMsgResolver = useFtlMsgResolver();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const searchParams = location.search;
+
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  let localizedErrorMessage = '';
+
+  const requestResetPasswordCode = async (email: string) => {
+    const metricsContext = queryParamsToMetricsContext(
+      flowQueryParams as unknown as Record<string, string>
+    );
+    const options = {
+      metricsContext,
+    };
+    try {
+      // TODO in FXA-7890 directly use auth client or not?
```

Comment:
I think for the PW redesign we opted to just use auth-client? I feel like if there's a state update or a value we need to reference elsewhere it's nice to use GQL so we don't have to manually add things to the apollo cache but not sure that's relevant for most reset PW stuff.